plotly/offline/offline.py

```diff
@@ -171,6 +184,34 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
                            height=height, width=width)))
 
 
+def iplot_mpl(mpl_fig,mpl_to_plotly_kw={},iplot_kw={}):
+    '''
+    Convert a matplotlib figure to plotly dictionary  plot inside an 
+    IPython notebook without connecting to an external server.
+    '''
```

Comment:
@arsenovic thanks for adding the docstrings! It might be good if they're expanded a little bit more (like https://github.com/arsenovic/plotly.py/blob/plotly_notebook_takeover/plotly/offline/offline.py#L82-L104) to include keyword argument descriptions and an example if applicable.